dao.py

```diff
@@ -1,34 +1,24 @@
-from flask_pymongo import PyMongo
-from model import Model
+from pymongo import MongoClient
 
 class Dao:
-    def __init__(self, app):
-        app.config['MONGO_DBNAME'] = 'thing_list'
-        
-        # TODO: clusert configuration is good here, but this will connect to a local mongo docker container once you build out
-        # your docker-compose file
-        app.config['MONGO_URI'] = 'mongodb://admin:admin@cluster0-shard-00-00-bwkql.mongodb.net:27017,cluster0-shard-00-01-bwkql.mongodb.net:27017,cluster0-shard-00-02-bwkql.mongodb.net:27017/test?ssl=true&replicaSet=Cluster0-shard-0&authSource=admin'
-        
-        self.mongo = PyMongo(app)
-
+    def __init__(self):
+        client = MongoClient('mongodb://admin:admin@cluster0-shard-00-00-bwkql.mongodb.net:27017,cluster0-shard-00-01-bwkql.mongodb.net:27017,cluster0-shard-00-02-bwkql.mongodb.net:27017/test?ssl=true&replicaSet=Cluster0-shard-0&authSource=admin')
+        self.db = client['thing_list']
 
     def add(self, item):
-        self.mongo.db['things'].insert_one(item.serialize())
+        self.db['things'].insert_one(item)
 
-    def find(self, name):
-        item = self.mongo.db['things'].find_one({'name':name})
+    def find(self, selector):
+        item = self.db['things'].find_one(selector)
         if not item:
             return None
         else:
-            return Model.from_dict(item)
+            return item
```

Comment:
`if not item` means item is None or False, simplified as
```py
return item or None
```

Reply:
fixed